README-local-dev.md

```diff
@@ -1,6 +1,8 @@
 # Quick start with PHP dev server
 
-The web2 site will run happily under the [PHP development server](http://php.net/manual/en/features.commandline.webserver.php), but you'll need to have a [Redis](http://redis.io) server running on localhost:6379.  You'll also need to configure a suitable [Joind.in API](https://github.com/joindin/joindin-api/); you could use the [live Joind.in API](https://api.joind.in/) or if you're developing against a local version, reference it in config/config.php.  Note that the API will also run under the built-in webserver, but will need to be on a different listening port.
+The web2 site will run happily under the [PHP development server](http://php.net/manual/en/features.commandline.webserver.php), but you'll need to have a [Redis](http://redis.io) server running on localhost:6379. You can easily do this with Docker typing `docker run --name some-redis -d -p 6379:6379 redis`.
```

Comment:
Let's try to avoid words like "easily" and "simply" since these are subjective and might not be as "easy" or "simple" to the end user as it is to the author.

```suggestion
The web2 site will run happily under the [PHP development server](http://php.net/manual/en/features.commandline.webserver.php), but you'll need to have a [Redis](http://redis.io) server running on localhost:6379. You can do this with Docker typing `docker run --name some-redis -d -p 6379:6379 redis`.
```